cg/apps/lims/api.py

```diff
@@ -93,25 +93,37 @@ def get_received_date(self, lims_id: str) -> str:
         """Get the date when a sample was received."""
 
         sample = Sample(self, id=lims_id)
-        return sample.udf.get("Received at") if sample else None
+        try:
+            return sample.udf.get("Received at")
+        except:
```

Comment:
Could you catch the expected error by importing in the top of the file:

```python
from requests.exceptions import HTTPError
```
Then 

```suggestion
        except HTTPError:
```
and same at the ones below